packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -356,42 +353,42 @@ test.describe('PieCookieBanner - Component tests', () => {
         test.describe('when not populated', () => {
             test('should set a default cookie statement link of empty string within the banner description', async ({ mount }) => {
                 // Arrange & Act
-                const component = await mount(
+                await mount(
                     PieCookieBanner,
                     {
                         props: {},
                     },
                 );
 
                 // Act
-                const cookieStatementLink = component.locator('[data-test-id="banner-description"] [data-test-id="cookie-statement-link"]');
+                const cookieBannerDescription = await pieCookieBannerComponent.getBannerCookieStatementLinkAttribute('href');
```

Comment:
This should stay as `cookieBannerLink`, or maybe `cookieBannerHref`, right?

Reply:
I think my point was this doesn't seem to be getting the description, as it's looking for the `href` attribute on a link, rather than the text content of a description. The other tests below that do similar all have `cookieBannerLink`, which I like.